pkg/operator/ceph/file/filesystem.go

```diff
@@ -140,9 +140,27 @@ func newFS(name, namespace string) *Filesystem {
 	}
 }
 
+// SetPoolSize function sets the sizes for MetadataPool and dataPool
+func SetPoolSize(f *Filesystem, context *clusterd.Context, spec cephv1.FilesystemSpec) error {
+	// generating the metadata pool's name
+	metadataPoolName := fmt.Sprintf("%s-%s", f.Name, metadataPoolSuffix)
```

Comment:
In the `doFilesystemCreate()` function the same formatters exist to generate the metadata and data pool names. Please refactor the name generation to common functions.

Reply:
Yes, I refactored it in the function down below.